alibuild_helpers/__init__.py

```diff
@@ -1,4 +1,7 @@
 # Dummy file to package build_template.sh
-import utilities as utilities
-import analytics as analytics
-import log as log
+
+from __future__ import absolute_import
+
+from . import utilities
```

Comment:
It looks like `absolute_import` was introduced as a `__future__` in Python2.5 (see [PEP 328](https://www.python.org/dev/peps/pep-0328/)). That's unfortunate the lowest target is 2.4.

I might be able to put that in an version_checking if block. Does anyone know if `from . import X` is a syntax error in 2.4.